ql-qlf-plugin/qlf_k6n10f/brams.txt

```diff
@@ -13,51 +12,21 @@ bram $__QLF_FACTOR_BRAM36_TDP
   dbits  2     @a14d2
   abits 15     @a15d1
   dbits  1     @a15d1
-  groups 2
-  ports  1 1
-  wrmode 0 1
-  enable 1 4   @a10d36
-  enable 1 2   @a11d18
-  enable 1 1   @a12d9 @a13d4 @a14d2 @a15d1
-  transp 0 0
-  clocks 2 3
-  clkpol 2 3
-endbram
-
-bram $__QLF_FACTOR_BRAM18_TDP
-  init 1
-  abits 10     @a10d18
-  dbits 18     @a10d18
-  abits 11     @a11d9
-  dbits  9     @a11d9
-  abits 12     @a12d4
-  dbits  4     @a12d4
-  abits 13     @a13d2
-  dbits  2     @a13d2
-  abits 14     @a14d1
-  dbits  1     @a14d1
-  groups 2
-  ports  1 1
-  wrmode 0 1
-  enable 1 2   @a10d18
-  enable 1 1   @a11d9 @a12d4 @a13d2 @a14d1
-  transp 0 0
-  clocks 2 3
-  clkpol 2 3
+  groups 4
+  ports  1 1 1 1
+  wrmode 0 1 0 1
+  enable 1 4 1 4 @a10d36
+  enable 1 2 1 2 @a11d18
+  enable 1 1 1 1 @a12d9 @a13d4 @a14d2 @a15d1
+  transp 0 0 0 0
+  clocks 1 1 2 2
+  clkpol 1 1 1 1
 endbram
 
 
 match $__QLF_FACTOR_BRAM36_TDP
-  min bits 128
-  min efficiency 2
-  shuffle_enable B
-  make_transp
-  or_next_if_better
-endmatch
-
-match $__QLF_FACTOR_BRAM18_TDP
-  min bits 128
-  min efficiency 2
+  max dbits 36
+  max abits 15
   shuffle_enable B
```

Comment:
1. There are no rules defined for 2 ports (1 Read, 1 Write) BRAM 
2. Thre are no rules defined for 18K BRAM. 36K BRAM can be split into 2 18K BRAM